src/server/scripts/Northrend/UtgardeKeep/UtgardePinnacle/boss_svala.cpp

```diff
@@ -159,14 +158,12 @@ class boss_svala : public CreatureScript
             {
                 _Reset();
 
-                SetCombatMovement(true);
-
                 if (_introCompleted)
                     events.SetPhase(NORMAL);
                 else
                     events.SetPhase(IDLE);
 
-                me->SetDisableGravity(false);
+                me->SetDisableGravity(false, true);
```

Comment:
this is a nope from me. Remove the true from here and from the part at line 373 and make sure that the boss has the no movement flag update extra flag (512)

Reply:
I have removed it in all four SetDisableGravity, If not, it was still falling after a wipe in ritual, removing it in all also works well, I hope this is the right way ;)